second-edition/src/ch07-01-mod-and-the-filesystem.md

```diff
@@ -293,12 +447,23 @@ about these warnings for now; we’ll address them later in this chapter in the
 “Controlling Visibility with `pub`” section. The good news is that they’re just
 warnings; our project built successfully!
 
+Esses warning(avisos) nos dizem que temos funções que nunca são usadas. Não se preocupe
```

Comment:
Acho que aqui já não precisa colocar **avisos** entre parênteses. Pode deixar apenas **warnings** (faltou o **s**, aliás). E como se trata de um estrangeirismo, é bom colocar em itálico, também.